pom.xml

```diff
@@ -78,14 +102,12 @@
                 <groupId>javax.enterprise</groupId>
                 <artifactId>cdi-api</artifactId>
                 <version>1.2</version>
-                <scope>provided</scope>
             </dependency>
             <dependency>
-                <groupId>javax.ws.rs</groupId>
-                <artifactId>javax.ws.rs-api</artifactId>
-                <version>2.0.1</version>
-                <scope>provided</scope>
-            </dependency>
+            	<groupId>javax.ws.rs</groupId>
+            	<artifactId>javax.ws.rs-api</artifactId>
+            	<version>2.0.1</version>
```

Comment:
this is fine, but just thought I'd ask whether we want to base on JAX-RS 2.1 instead of 2.0.  thoughts?  Without 2.1, we won't be able to handle SSE events.

Reply:
JAX-RS 2.1 is not in MicroProfile as yet. If Rest client is released in MP 1.3, it needs to work with JAX-RS 2.0. In MP 2.0, 1Q next year, it can use JAX-RS 2.1.